### ISSUE TITLE
OAuth / OpenIDConnect support

### ISSUE THREAD

**Issue by jwillmer** [elsa-workflows/elsa-core#2465]
As a user I like to have a global credential manager there I can setup OAuth connections and use them in my workflow activities.

**Comment by tomy2105** [comment]
You are talking about adding support for, for example, HTTP Send Request Activity to be able to use OAuth/OIdC, right?
Or and HTTP Endpoint implementing OAuth/OIdC based authorization?

Come to think of it, where are with multitenancy and adding user authentification layer to UI (which could be OAuth/OIdC based as one possibility)?

**Comment by jwillmer** [comment]
Yes, we need to connect to many systems that use OAuth. So we should support OAuth in the HTTP activity.

**Comment by LinQiaoPorco** [comment]
https://docs.elsaworkflows.io/authentication/authentication#using-oidc-openid-connect

Can we give more details & samples in the doc? I did not quite follow this guide.
